mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
+      if (error) {
```

Comment:
Inconsistent error handling. The error object is handled differently in various parts of the code. In some cases, it's checked for existence and then its properties are accessed, while in others, it's directly accessed without checking. This inconsistency can lead to unexpected behavior or errors.